content/en/docs/ops/common-problems/security-issues/index.md

```diff
@@ -84,17 +84,15 @@ matching requests should flow through. If all requests continue to be denied, yo
 
 1. Make sure there is no typo in your policy YAML file.
```

Comment:
This seems unhelpful. The yaml parser and analyzer should complain about many typos. I would suggest instead describing specific common typos that can slip through, or remove this item. Just saying "typo" is too general IMO.

Reply:
I think the yaml parser and the analyzer doesn't catch the typos, for example, if I apply the following policy:
```yaml
apiVersion: security.istio.io/v1beta1
kind: AuthorizationPolicy
metadata:
 name: test
 namespace: foo
spec:
 action: ALLOW
 selector:
   matchLabels:
     app: test
 typo_rules:
 - from:
   - source:
       principals: ["cluster.local/ns/istio-system/sa/istio-ingressgateway-service-account"]
```

the `typo_rules` field will not be caught and the policy will be accepted. The unknown fields will just be ignored when converting to the AuthorizationPolicy protobuf in Pilot.